cloudbuild.yaml

```
@@ -58,12 +58,48 @@ steps:
       - cp
       - output/trusted_applet.sig
       - gs://${_TRUSTED_APPLET_BUCKET}/${TAG_NAME}/trusted_applet.sig
-
+      # Construct log entry / Claimant Model statement.
+  - name: 'ubuntu'
+    entrypoint: 'bash'
+    args:
+      - '-c'
+      - |
+        cp output/trusted_applet.elf . && \
+        echo -e "{\n  digest: $(sha256sum -b trusted_applet.elf),\n  tag_name: ${TAG_NAME},\n  tamago_version: ${_TAMAGO_VERSION},\n}" > output/applet_log_entry
```

You probably only want the digest and not the filename in here?

```suggestion
        echo -e "{\n  digest: $(sha256sum -b trusted_applet.elf | cut -d " " -f1),\n  tag_name: ${TAG_NAME},\n  tamago_version: ${_TAMAGO_VERSION},\n}" > output/applet_log_entry
```

The current output contains the filename `digest: 4b832434ff1d403eb32d8b265647240bdc68e3e22e3323edf91d2907cc08f81b *output/trusted_applet.elf,`

---

Done.